### PR TITLE
Fix markup issues and warnings in nf.mm

### PR DESCRIPTION
**Pull request by digama0**
Mostly just rewrapping and adding `New usage is discouraged.` on OLD and ALT theorems. I also had to add the garden path directive from set.mm, I just put it next to the typesetting section since @tirix doesn't have a mathbox here.

### CODE REVIEW

**Review by jkingdon — APPROVED** [review]

**Review by benjub — APPROVED** [review]
LGTM, but we should probably wait for @scott-fenton or @sctfn's approval.

**Review by tirix — APPROVED** [review]

**Comment by sctfn** [comment]
LGTM

**Pull request merged**
